app.js

```
@@ -124,315 +129,1806 @@ function getRiskLevel(points) {
+      channel: channelId,
+      text: `<@${userId}> invited ${carrierName} (MC${mcNumber}) via Intellivite\nContact: ${safeEmail}`,
+    });
+  } catch (error) {
+    logger.error({ err: error }, "Failed to post success message");
+  }
+});
+
+// Handle Step 4 modal close
+slackApp.view(
+  { callback_id: "carrier_wizard_step4", type: "view_closed" },
+  async ({ ack, body, view }) => {
```

ESLint

[error] 'body' is defined but never used.

(no-unused-vars)